README.md

```diff
@@ -1,14 +1,86 @@
-# React sc-web
+## React sc-web
 
-### Quick start (recommended)
+- [Quick start](#quick-start)
+- [Installation](#installation)
+  * [Production](#production)
+  * [Development](#development)
+- [Develop with Docker in another project](#develop-with-docker-in-another-project)
+- [Configuration](#configuration)
+- [Recommendations](#recommendations)
+- [Files structure](#files-structure)
+
+### Quick start
 
 Go to the [OSTIS Metasystem](https://github.com/ostis-ai/ostis-metasystem), checkout to the `demo` branch and follow installation and run instructions.
 
-### Run in another system
+### Installation
+
+First of all **clone the project on you local machine**.
+
+```sh
+git clone git@github.com:ostis-ai/react-sc-web.git
+cd react-sc-web
+```
+
+Note that this project has runtime dependency on [sc-web](https://github.com/ostis-ai/sc-web), which must be up and running with **allowed origins**.

```

Comment:
```suggestion
Note that this project has runtime dependency on [sc-web](https://github.com/ostis-ai/sc-web), which must be up and running with **allowed origins** (python3 sc-web/server/app.py" --allowed_origins=http://localhost:3000).

```

Reply:
Well, this information is given below. I don't think that there is a lot of sense in this line here. What do you think?